server/cmwell-spa/utils/upload.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+tokenFileName=~/dev/uploadTool/token-$1
+[ -e "$tokenFileName" ] || touch $tokenFileName -d 2000-01-01
+
+tokenAge=$(($(date +%s) - $(date +%s -r $tokenFileName)))
+if [ $tokenAge -gt 3600 ]
+then
+    token=$(curl -s "http://${1:-localhost:9000}/_login" -u root | jq -r ".token")
+    echo $token > $tokenFileName
+else
+    token=$(cat $tokenFileName)
+fi
+
+# Create AppCache Manifest according to file list and commit hash
+
+commit=$(git rev-parse HEAD)
+fileList=$(find -type f | grep -vE 'index.html|cmwell.appcache' | sed 's/^\./\/meta\/app\/main/')
+echo -e "CACHE MANIFEST\n\n# $commit\n\nCACHE:\n$fileList\n\nNETWORK:\n*" > cmwell.appcache
```

Comment:
Continuing the global `~/.cmwell/` dir thought, such files like appcache should be stored there

Reply:
I removed this part from this util. see [this commit](https://github.com/thomsonreuters/CM-Well/pull/171/commits/23d340a93217a8992383fecb562e6cb675a9e88f)